public/app/features/datasources/DataSourcesListItem.tsx

```diff
@@ -1,35 +1,151 @@
 import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import appEvents from 'app/core/app_events';
+import { deleteDataSource, loadDataSource, loadDataSources } from './state/actions';
+import { StoreState, CoreEvents } from 'app/types/';
 import { DataSourceSettings } from '@grafana/ui';
 
 export interface Props {
   dataSource: DataSourceSettings;
+  loadDataSources: typeof loadDataSources;
+  loadDataSource: typeof loadDataSource;
+  deleteDataSource: typeof deleteDataSource;
 }
 
-export class DataSourcesListItem extends PureComponent<Props> {
+export interface State {
+  hover: boolean;
+  deleteConfirmOpen: boolean;
+}
+
+export class DataSourcesListItem extends PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      hover: false,
+      deleteConfirmOpen: false,
+    };
+  }
+
+  enterHover = () => {
```

Comment:
we follow AirBnb react style guide which means event handles should follow same naming as event name, so this would be onMouseEnter, orHoverEnter
https://github.com/grafana/grafana/blob/master/contribute/style-guides/frontend.md#props

Reply:
fixed